pkgs/cpp-semver/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "cpp-semver";
+  version = "unstabble-2021-12-10";
+
+  src = fetchFromGitHub {
+    owner = "easz";
```

Comment:
blocking question (to at least discuss): are we comfortable with how well-maintained this repo is? No commits for 3 years, and no response to @tomberek's PR in 2 months https://github.com/easz/cpp-semver/pull/8

How do https://github.com/Neargye/semver or  https://github.com/z4kn4fein/cpp-semver compare? Are any of them heavily used?

Reply:
Discussed in the team meeting briefly.  I think we're OK for now based on the following factors:
- There don't seem to be _easy_ alternatives.  There are library based replacements, but that would be more complicated to integrate.  Other header only alternatives don't appear to have the functionality this repo does (ranges, etc).
- I think we would prefer to include this filtering directly in the SQL queries so this is not likely to be too long lived.
- The feature set is for a well defined standard that is not in flux and is likely pretty static. So there is likely not too much benefit from on going maintenance.

For these  reasons, I think the consensus is to proceed as is.  Any final objections or concerns?

Reply:
Sounds good to me 👍